src/platform/test.rs

```diff
@@ -775,6 +775,7 @@ mod sync_test {
 // cross_process_sender_transfer_spawn() test below.  Running it by
 // itself is meaningless.
 #[cfg(not(any(feature = "force-inprocess", target_os = "android")))]
+#[cfg_attr(target_os = "windows", should_panic(expected = "received handles intended for process"))]
```

Comment:
I'll just mark it as ignored -- if it's good enough to mark this as ignored and put a comment in, it should be good enough to mark the "server" tests as ignored with another comment.  Those tests pass when they are run by themselves as well.